types/react-native-snap-carousel/index.d.ts

```diff
@@ -46,6 +57,16 @@ export interface CarouselProps extends React.Props<ScrollViewProperties> {
      * From slider's center, minimum slide distance to be scrolled before being set to active
      */
     activeSlideOffset?: number;
+    /**
+     * Duration of time while component is hidden after mounting. NOTE: May cause rendering
+     * issues on Android. Defaults to 0
+     */
+    apaparitionDelay?: number;
```

Comment:
This seems to be typo: `apparitionDelay`

Reply:
Thanks for catching!